site/assets/scss/_clipboard-js.scss

```diff
@@ -1,8 +1,8 @@
 // clipboard.js
 //
-// JS-based `Copy` buttons for code snippets.
+// JS-based `Copy` and `Try on Stackblitz` buttons for code snippets.
 
-.bd-clipboard {
```

Comment:
The CSS changes here need review and they probably don't match the filename (_clipboard-js.scss)

Reply:
All these need to be reverted